File: CinemaProject.WebAPI/CinemaProject.BLL/Models/Film.cs
using CinemaProject.BLL.Helpers;
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace CinemaProject.BLL.Models
{
    public class Film
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Country { get; set; }
        public DateTime ReleaseDate { get; set; }
        [JsonConverter(typeof(TimeSpanJsonConverter))]
        public TimeSpan Duration { get; set; }
        public string Director { get; set; }

        public IList<Actor> Actors { get; set; }
        public IList<Genre> Genres { get; set; }
    }
}

File: CinemaProject.Angular/src/app/AdminPages/Models/film.ts
export class Film {
  constructor(
    public id: string,
    public name: string,
    public country: string,
    public releaseDate: Date,
    public duration: string,
    public director: string
  ) {}
}

export class FilmView {
  constructor(
    public id: number,
    public name: string,
    public country: string,
    public releaseDate: string,
    public duration: string,
    public director: string,
    private guidId: string
  ) {}
}
